src/algebraic_geometry/structure_sheaf.lean

```diff
@@ -229,6 +312,42 @@ def structure_sheaf : sheaf CommRing (Spec.Top R) :=
       (sheaf_condition_equiv_of_iso (structure_presheaf_comp_forget R).symm
         (structure_sheaf_in_Type R).sheaf_condition), }
 
--- TODO: we need to prove that the stalk at `P` is `localization.at_prime P.as_ideal`
+/--
+The stalk at `x` is equivalent (just as a type) to the localization at `x`.
+-/
+def stalk_iso_Type (x : prime_spectrum R) :
+  (structure_sheaf_in_Type R).presheaf.stalk x ≅ localization.at_prime x.as_ideal :=
+(equiv.of_bijective _
+  ⟨structure_sheaf_stalk_to_fiber_injective R x,
+   structure_sheaf_stalk_to_fiber_surjective R x⟩).to_iso
+
+-- PROJECT: Improve this to an isomorphism of rings.
```

Comment:
ditto

Reply:
I propose leaving this for a future PR --- you should do it! :-)